README.md

```diff
@@ -73,6 +73,7 @@ For more detailed information, see the [White Paper](https://www.iresty.com/down
 - **CLI**: start\stop\reload APISIX through the command line.
 - **REST API**
 - **Proxy Websocket**
+- **Response Rewrite**: Set customized body&header to client, support CORS,Redirect.
```

Comment:
I think APISIX already has `Redirect` plugin, and `CORS` is also a separate plugin.
So `Set customized response body and header to the client` is good enough.

Reply:
ok，how about change it into `Set customized response status code, body and header to the client.`

Reply:
Looks good to me.